roles/vendor/opensearch/tasks/opensearch.yml

```diff
@@ -57,9 +57,10 @@
       group: "{{ os_user }}"
       mode: "u=rwX,g=rwX"
       remote_src: true
+    notify:
```

Comment:
it is impossible to use 'notify' in a 'block' segment. I think it should be there?